src/unstructured_client/_hooks/custom/split_pdf_hook.py

```diff
@@ -5,26 +5,26 @@
 import io
 import json
 import logging
-import os
-from concurrent.futures import ThreadPoolExecutor, Future
-from typing import Optional, Tuple, Union, Generator
+import math
+from concurrent.futures import Future, ThreadPoolExecutor
```

Comment:
this could just be me overthinking:
- threadpool can trip on GIL and lead to unintended thread lock
- process pool provides process isolation
- we don't actually share data between threads so there is no advantage to have them under one process
- so a process pool would be safer option here

Reply:
yeah, I'll change it to `ProcessPoolExecutor`